jobs/build/reposync/Jenkinsfile

```diff
@@ -15,6 +16,13 @@ node {
                 $class: 'ParametersDefinitionProperty',
                 parameterDefinitions: [
                     commonlib.ocpVersionParam('SYNC_VERSION'),
+                    [
+                        name: 'ARCH',
+                        description: 'Architecture of repo to synchronize; must have full definition in group.yml for release',
+                        $class: 'hudson.model.ChoiceParameterDefinition',
+                        choices: [ 'x86_64', 'ppc64le' ].join("\n"),
```

Comment:
```suggestion
                        choices: [ 'x86_64', 'ppc64le', 's390x' ].join("\n"),
```

Reply:
Added. Just a note that any attempt to run with this arch will fail (safely, but loudly) until an s390x url is added for each repo.